deployment/ansible/inventory/production.j2

```diff
@@ -0,0 +1,17 @@
+app ansible_ssh_host={{ app_server_ip }}
```

Comment:
In our most common use-case, which is EC2 instances in a VM, these IP addresses are different from the IP addresses used in the group_vars. For example, for the Mumbai instance, the group_vars IPs look like this:
- `app_server_ip`: `172.31.25.22`
- `celery_server_ip`: `172.31.26.26`
- `database_server_ip`: `172.31.24.80`

But the inventory / public IPs are different; the App Server's public IP is `54.148.118.47`, for example. 

On the other hand, for the Philippines deployment, each machine has only one IP, the public IP.

I think there are a couple ways to handle this. We could add a branch to the wizard, which would prompt the user with a question like "Are you running your servers in a virtual private cloud?" and then additionally prompt for different IPs if so. Alternatively, we could leave this as-is, and just add a note after the wizard finishes saying something like "If your servers have different internal and external IP addresses, you'll need to edit `deployment/ansible/inventory/production` to add the proper external addresses."

I'm leaning toward the second one because there is no guarantee that the instances will have publicly-available IPs at all; a more secure way to administer the app would be not to allocate any public IPs and to just administer everything from a "bastion" instance that has access to the inside of the VPC.

Reply:
How about if we prompt for public IPs for each server and you can leave it blank if it's n/a? That would work for our most common use case.

I think for now it makes sense to make this wizard work for you and Kenny, the people actually doing the deployments, so we should cater to our common use case without totally mystifying (and hopefully helping) anyone deploying DRIVER for the first time.

Reply:
Yeah, that seems reasonable, we can just note that it's okay to leave those blank.